fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -763,6 +763,23 @@ private String mintNewPid(final String slug) {
         return pid;
     }
 
+    /**
+     * perform save with checking to prevent SN-Sibling nodes from creating
+     * @param resource the FedoraResource need to save
+     * @throws RepositoryException
+     */
+    private void save(final FedoraResource resource) throws RepositoryException {
+        synchronized (LOGGER) {
```

Comment:
I'm with @ajs6f -- the synchronization bit needs to go. Imagine what this would look like in a distributed context--it wouldn't even approach a workable solution. If you need some form of process coordination, you need something more than a static LOGGER variable.